components/conjunctions/index.json-ld

```diff
@@ -14,7 +14,7 @@
 		"en": "Conjunctions for stacked or side by side decision points, and/or",
 		"fr": "Conjonctions pour des points de décision empilés ou côte à côte, et/ou"
 	},
-	"modified": "2025-07-14",
+	"modified": "2025-10-03",
```

Comment:
This would be version 1.0.1 as this is a patch change. Please also add a new iteration with documented change in "fixes" attribute.